### PR TITLE
Batch Processing and Docker Support

### PR DESCRIPTION
**Pull request by knrdl**
Hi, first of all thanks a lot for this awesome project!
It saved me a lot of time and effort, blurring dashcam videos showing maniac car drivers disregarding rules on purpose and endangering me as well as other cyclists. 
I use the software to batch process videos inside of a podman (/docker) container.
This setup might be useful for others too, so I wanted to share it.

### CODE REVIEW

File: dashcamcleaner/src/blurrer.py [hunk 1]
```diff
@@ -85,20 +85,22 @@ def apply_blur(self: 'VideoBlurrer', frame: np.array, new_detections: List[Detec
...
+                single_detection_mask_color = (mask_color,) * 3
 
                 detection_mask = np.full((frame.shape[0], frame.shape[1], 3), 0, dtype=np.float64)
                 # add detection bounds to mask
                 if detection.kind == "plate":
                     if export_colored_mask and detection.score:
                         single_detection_mask_color = (0, mask_color * detection.score, 0)
-                    cv2.rectangle(detection_mask, bounds.pt1(), bounds.pt2(), color=single_detection_mask_color, thickness=-1)
+                    cv2.rectangle(
+                        detection_mask, bounds.pt1(), bounds.pt2(), color=single_detection_mask_color, thickness=-1)
```
Comment (joshinils) [inline dashcamcleaner/src/blurrer.py:96]: i guess this is due to some linter rule of your editor?

Comment (knrdl) [reply dashcamcleaner/src/blurrer.py:96]: Yes, i guess IntelliJ doesn't like it when lines got too long

Comment (knrdl) [reply dashcamcleaner/src/blurrer.py:96]: There is indeed a single, real code change: https://github.com/tfaehse/DashcamCleaner/pull/47/commits/589bcce0d673e6096568ac9c5ce78f03e9a7c8c0#diff-dc15d6e41ef6c6b5e2ae45f31eb3dd289b44ea277afa9438d0d8e4cb1ee90090R140
An iterable is expected, the original code did not run for me with `size=scale`, but `size=(scale,)`

**Comment by tfaehse** [comment]
Wow, thank you for the contribution! This looks absolutely amazing, I'll do a (very) quick test and check the implementation (mainly out of curiosity) in 1-2 days and merge it, I'm super happy to see more and more really valuable contributions here!

File: dashcamcleaner/src/blurrer.py (hunk 1 above)
Comment (tfaehse) [reply dashcamcleaner/src/blurrer.py:96]: Now that's confusing. If I run your code it fails with the following trace:
```
File "/Users/tfaehse/Developer/dashcamcleaner/src/blurrer.py", line 140, in detect_identifiable_information
    results_list = self.detector(images, size=(scale,)).xyxy
  File "/Users/tfaehse/miniforge3/envs/py38/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/Users/tfaehse/miniforge3/envs/py38/lib/python3.8/site-packages/torch/autograd/grad_mode.py", line 28, in decorate_context
    return func(*args, **kwargs)
  File "/Users/tfaehse/.cache/torch/hub/ultralytics_yolov5_master/models/common.py", line 611, in forward
    g = (size / max(s))  # gain
TypeError: unsupported operand type(s) for /: 'tuple' and 'int'
```

Funny, torch/yolov5 seems to have changed that very recently. I deleted by torch cache and now it works fine with yours, and the current main doesn't work... Thanks for the fix, I guess :D

**Pull request merged**